### PR TITLE
Mc knight 42/update contributing

### PR DESCRIPTION
**Pull request by McKnight-42**
…pelines to Github Actions for CI testing.

resolves #https://github.com/dbt-labs/core-team/issues/29

### Description

 - Updated Contribute.MD to reflect swap over from CircleCI and Azure Pipelines to Github Actions.
 - Made Changes to reflect Removal of Adapaters from dbt-core repo to own repos
 - Added link and message to convey that Adapter specific issues/contributions should be made on their repos not the core repo.

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [ ] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have updated the `CHANGELOG.md` and added information about my change to the "dbt next" section.

### CODE REVIEW

**Comment by McKnight-42** [comment]
@cla-bot check

**Review by kwigley — APPROVED** [review]
Looks good to me 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -103,7 +107,7 @@ A short list of tools used in `dbt` testing that will be helpful to your underst
 - [`make`](https://users.cs.duke.edu/~ola/courses/programming/Makefiles/Makefiles.html) - but don't worry too much, nobody _really_ understands how make works and our Makefile is super simple
 - [`flake8`](https://flake8.pycqa.org/en/latest/) for code linting
 - [`mypy`](https://mypy.readthedocs.io/en/stable/) for static type checking
-- [CircleCI](https://circleci.com/product/) and [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/)
+- [Github Actions](https://github.com/features/actions)
```
Comment (jtcohen6) [inline CONTRIBUTING.md:110]: 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -226,7 +230,7 @@ python -m pytest test/unit/test_graph.py::GraphTest::test__dependency_list
 > is a list of useful command-line options for `pytest` to use while developing.
 ## Submitting a Pull Request
 
-dbt Labs provides a sandboxed Redshift, Snowflake, and BigQuery database for use in a CI environment. When pull requests are submitted to the `dbt-labs/dbt` repo, GitHub will trigger automated tests in CircleCI and Azure Pipelines.
+dbt Labs provides prodives a CI environment to test changes to specific adapters and periodic maintence checks of `dbt-core` through Github Actions. When pull requests are submitted to the `dbt-labs/dbt-redshift` repo for example these test will fire.
```
Comment (jtcohen6) [inline CONTRIBUTING.md:233]: The idea here is right on. How about something like:
```suggestion
dbt Labs provides a CI environment to test changes to specific adapters, and periodic maintenance checks of `dbt-core` through Github Actions. For example, if you submit a pull request to the `dbt-redshift` repo, GitHub will trigger automated code checks and tests against Redshift.

**Review by jtcohen6 — APPROVED** [review]
Nice work @McKnight-42! I left one suggestion. After addressing, feel free to merge without waiting for another review.

**Pull request merged**